modules/install-sno-ibm-z-lpar.adoc

```diff
@@ -0,0 +1,188 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-sno-on-ibm-z-lpar_{context}"]
+= Installing {sno} in an LPAR on {ibm-z-title} and {ibm-linuxone-title}
+
+.Prerequisites
+
+* You have installed `podman`.
+* You have an existing `install-config.yaml` file.
```

Comment:
Why do we require to have an existing "install-config.yaml" ?
Because, It is described below how to "Prepare the `install-config.yaml` file"
I would remove this line.